### DISCUSSION TITLE
Questions about FIADB

### DISCUSSION THREAD

**Discussion by diazrenata** [mekevans/forestTIME#24] [General]
This is a thread for questions about FIADB.

**Comment by diazrenata** [comment]
I often encounter situations where the same plot/tree will have two records, with different CN numbers, that differ only in their CONDITION attributes (COND_STATUS_CD, CONDID, OWNCD).

**Reply by mekevans** [reply]
this is exactly what Ceci is finding in the WBP data set! We need to figure this out

**Reply by mekevans** [reply]
I don't understand how the same tree in the same year can belong to two different conditions

**Comment by Kah5** [comment]
Just checking in about this--it sounds like these are from trees from the
same inventory year, are trees on the same subplot (SUBP)?

**Reply by diazrenata** [reply]
I think it might have to do with the CONDITION table having multiple records per PLT_CN. I put a summary doc here about what I (think I'm) seeing: https://github.com/mekevans/forestTIME/blob/main/use_cases/repeat_cns/repeat_cns.md

Because there are multiple CONDITION records per PLT_CN, when you join the CONDITION and TREE tables by PLT_CN you end up with multiple records per tree with different CONDITION variables. 

I think this might be because a plot can have multiple conditions. Should we maybe be using the SUBPLOT_CONDITION table?

**Comment by Kah5** [comment]
Thanks for putting this table together, this is helpful. Each plot can have
multiple conditions, and so can each subplot. According to the manual,
there should be a CONDID column for each tree in the TREE table, and this
CONDID, along with PLT_CN, SUBP should match up to the CONDITION table to
give more information on what the CONDID means. I am not 100% sure on
whether we should be using the SUBPLOT_CONDITON or just the CONDITON table,
but I can send Brian Walters a message about this.

**Comment by mekevans** [comment]
I wrote to Brian Walters today and got this response: 
to answer Renata’s question: it appears that you are getting duplicate trees because of an incomplete join. The tree table should be joined to the condition table using PLT_CN and CONDID. If you only use PLT_CN then each tree will join to each condition that’s on the plot. When there is a plot with multiple conditions you’ll get duplicate tree results.

**Comment by diazrenata** [comment]
Ah, fantastic! For some reason I thought the CONDID described something qualitative and didn’t realize it was a matching key. Thanks to everyone for helping sleuth this out!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: mekevans ***@***.***>
Sent: Thursday, February 1, 2024 6:14:18 PM
To: mekevans/forestTIME ***@***.***>
Cc: Diaz, Renata - (renatadiaz) ***@***.***>; Author ***@***.***>
Subject: [EXT]Re: [mekevans/forestTIME] Questions about FIADB (Discussion #24)


External Email

I wrote to Brian Walters today and got this response:
to answer Renata’s question: it appears that you are getting duplicate trees because of an incomplete join. The tree table should be joined to the condition table using PLT_CN and CONDID. If you only use PLT_CN then each tree will join to each condition that’s on the plot. When there is a plot with multiple conditions you’ll get duplicate tree results.

**Comment by diazrenata** [comment]
I believe I've fixed this (see #25). We can talk about this when we meet!
